docker-compose.yml

```
@@ -0,0 +1,311 @@
+    container_name: minio
+    command: server /data --console-address ":9001"
+    environment:
+      MINIO_ROOT_USER: "${MINIO_ROOT_USER}"
+      MINIO_ROOT_PASSWORD: "${MINIO_ROOT_PASSWORD}"
+      MINIO_ACCESS_KEY: minioadmin
+      MINIO_SECRET_KEY: minioadmin
+    networks:
+      - mirror-node-bridge
+    ports:
+      - "9000:9000"
+      - "9001:9001"
+    volumes:
+      - "${MINIO_DATA_PATH}:/data"
+
+  db:
+    image: "${MIRROR_POSTGRES_IMAGE}"
+    container_name: mirror-node-db
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: scram-sha-256
+      POSTGRES_INITDB_ARGS: --auth-host=scram-sha-256
+      GRPC_PASSWORD: mirror_grpc_pass
+      IMPORTER_PASSWORD: mirror_importer_pass
+      OWNER_PASSWORD: mirror_node_pass
+      POSTGRES_PASSWORD: postgres_password
+      REST_PASSWORD: mirror_api_pass
+      ROSETTA_PASSWORD: mirror_rosetta_pass
+    networks:
+      - mirror-node-bridge
+    expose:
+      - "5432"
+    restart: unless-stopped
+    stop_grace_period: 2m
+    stop_signal: SIGTERM
+    tty: true
+    volumes:
+      - ./compose-network/mirror-node/init.sh:/docker-entrypoint-initdb.d/init.sh
+      - ./compose-network/mirror-node/application.yml:/usr/etc/hedera-mirror-importer/application.yml
+      - ./compose-network/mirror-node/addressBook.bin:/usr/etc/hedera-mirror-importer/local-dev-1-node.addressbook.f102.json.bin
+
+  grpc:
+    image: gcr.io/mirrornode/hedera-mirror-grpc:0.49.0
+    container_name: mirror-node-grpc
+    restart: unless-stopped
+    mem_swappiness: 0
+    mem_limit: 512m
+    memswap_limit: 512m
+    depends_on:
+      network-node:
+        condition: service_started
+      db:
+        condition: service_started
+    environment:
+      HEDERA_MIRROR_GRPC_DB_HOST: db
+      SPRING_CONFIG_ADDITIONAL_LOCATION: file:/usr/etc/hedera-mirror-grpc/
+    networks:
+      - mirror-node-bridge
+    ports:
+      - "5600:5600"
+    volumes:
+      - ./compose-network/mirror-node/application.yml:/usr/etc/hedera-mirror-grpc/application.yml
+
+  importer:
+    image: gcr.io/mirrornode/hedera-mirror-importer:0.49.0
+    container_name: mirror-node-importer
+    restart: unless-stopped
+    mem_swappiness: 0
+    mem_limit: 768m
+    memswap_limit: 768m
+    depends_on:
+      network-node:
+        condition: service_started
+      db:
+        condition: service_started
+    environment:
+      HEDERA_MIRROR_IMPORTER_DB_HOST: db
+      SPRING_CONFIG_ADDITIONAL_LOCATION: file:/usr/etc/hedera-mirror-importer/
+    networks:
+      - mirror-node-bridge
+    volumes:
+      - ./compose-network/mirror-node/application.yml:/usr/etc/hedera-mirror-importer/application.yml
+      - ./compose-network/mirror-node/addressBook.bin:/usr/etc/hedera-mirror-importer/local-dev-1-node.addressbook.f102.json.bin
+
+  rest:
+    image: gcr.io/mirrornode/hedera-mirror-rest:0.49.0
+    container_name: mirror-node-rest
+    mem_swappiness: 0
+    mem_limit: 512m
+    memswap_limit: 512m
+    depends_on:
+      network-node:
```

Doesn't depend upon network-node. Should depend upon importer instead since he creates the schema.
```suggestion
      importer:
```

---

depend on `network-node` is intentionally added to make sure consensus node is started first , before any mirror node components, to resolve some error.  Modified to depend on `importer` , will re-test multiple times to see if no issues occur.